### PR TITLE
Added docs to Scene_Base

### PR DESCRIPTION
**Pull request by niokasgami**
Added documentations to the Scene_Base class.

### CODE REVIEW

**Review by ghost — APPROVED** [review]

**Comment by ghost** [comment]
We'll remove version info, because It is need in _header.js only. Is it ok?

**Comment by ghost** [comment]
wether => whether?

**Comment by niokasgami** [comment]
@liply Sure! I am sorry I forgot to answer was busy with stuff!
Yes it's could be put into the Header!

as for the Whether sorry it's was a misspelling of myself do you want me to fix it?

**Comment by ghost** [comment]
could you fix it?

**Comment by niokasgami** [comment]
@liply : Sure! 
here you go it's fixed! 
I was adding the header since it's was separating the class but I guess it's less JS-likes hahahah

File: js/rpg_scenes/Scene_Base.js [hunk 1]
```diff
@@ -19,45 +30,128 @@ Scene_Base.prototype.initialize = function() {
...
+ * Update the scene processing each new frame.
+ * 
+ * @method update
+ * @instance 
+ * @memberof Scene_Base
+ */
 Scene_Base.prototype.update = function() {
     this.updateFade();
     this.updateChildren();
+    AudioManager.checkErrors();
```
Comment (krmbn0576) [inline js/rpg_scenes/Scene_Base.js]: I removed this line before.
Could you remove?

Comment (niokasgami) [reply js/rpg_scenes/Scene_Base.js]: Sure I guess my version was a little outdated!

**Review by krmbn0576 — APPROVED** [review]
Nice! ❤️

**Pull request merged**